code/datums/quirks/positive_quirks.dm

```diff
@@ -265,6 +298,17 @@
 /datum/quirk/item_quirk/tagger/add_unique(client/client_source)
 	give_item_to_holder(/obj/item/toy/crayon/spraycan, list(LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
 
+/datum/quirk/throwingarm
+	name = "Throwing Arm"
+	desc = "Your arms have a lot of heft to them! Objects that you throw just always seem to fly further than everyone elses, and you never miss a toss."
+	icon = "baseball"
+	value = 2
```

Comment:
extra throw range actually has a ton of value, 2 is way too low. something like a 6-8 would be better